decision_service/rlclientlib/live_model_impl.cc

```diff
@@ -31,26 +46,29 @@ namespace reinforcement_learning
     api_status::try_clear(status);
 
     //check arguments
-    TRY_OR_RETURN(check_null_or_empty(uuid, context, status));
-
-    /* GET ACTIONS PROBABILITIES FROM VW */
-
-    /**/ // TODO: replace with call to parse example and predict
-    /**/ // TODO: once that is complete
-    response.push_back(2, 0.4f);
-    response.push_back(1, 0.3f);
-    response.push_back(4, 0.2f);
-    response.push_back(3, 0.1f);
-      
-    const string model_id = "model_id";
-      
-    //send the ranking event to the backend
-    _buff.seekp(0, _buff.beg);
-    ranking_event::serialize(_buff, uuid, context, response, model_id);
-    auto sbuf = _buff.str();
-    TRY_OR_RETURN(_logger.append_ranking(sbuf, status));
+    RETURN_IF_FAIL(check_null_or_empty(uuid, context, status));
+
+    int scode;
+    if(!_model_data_received) {
```

Comment:
>if(!_model_data_received) { [](start = 4, length = 27)

I assume this is set after the model is set, right?

Reply:
Yes.  set to false initially and then set to true first time data is received and model is successfully hydrated.